CodeSniffer/File.php

```diff
@@ -1482,6 +1490,7 @@ private static function _createPositionMap(&$tokens, $tokenizer, $eolChar, $enco
         $tokenizerType = get_class($tokenizer);
         $ignoring      = false;
         $inTests       = defined('PHP_CODESNIFFER_IN_TESTS');
+        $tabLines      = [];
```

Comment:
You can't use short array syntax in version 2.x of PHPCS. I can fix this after merging, but you might be able to get to it before I do.